account_invoice_validate_send_email/models/mail_compose_message.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailComposeMessage(models.TransientModel):
+    _inherit = "mail.compose.message"
+
+    @api.multi
+    def get_mail_values(self, res_ids):
+        self.ensure_one()
+        results = super(MailComposeMessage, self).get_mail_values(res_ids)
+        for res_id in res_ids:
+            if self.reply_to and not results[res_id].get("reply_to", False):
```

Comment:
```suggestion
            if self.reply_to and not "reply_to" in results[res_id]:
```
> I push a commit. I guess `if self.reply_to and not results[res_id].get("reply_to", False):` (at L16) would be appropriate to limit the scope.

You are right, and considering the case `results[reply_to]` is set to `False` for whatever reason (not sure if it ever happens), we may simply check if the key exists in the dict (if the key exist in the dict we should respect it). What do you think?

Reply:
definitely, important things rely on the existence of results[res_id]["reply_to"] value, so I agree that simple key check is enough.
In addition, `"reply_to" in results[res_id]` is more readable, I felt strongly.